cfpb/consumerfinance.gov
# CCDB landing: Add aria-labels to "Learn more" links

"Learn more" links were not providing enough context to screen readers that were browsing a list of only page links.

## Changes

- CCDB landing: Add aria-labels to "Learn more" links


## How to test this PR

1. Pull branch and visit http://localhost:8000/data-research/consumer-complaints/ and compare to https://www.consumerfinance.gov/data-research/consumer-complaints/
2. Turn on voiceover and see that the "Learn more" links in this PR are more descriptive than the ones on production, which only say "Learn more". Labels are also visible with WebAIM extension.


## Screenshots

<img width="827" alt="Screen Shot 2021-06-21 at 4 39 24 PM" src="https://user-images.githubusercontent.com/704760/122825346-cfa8fb80-d2af-11eb-8d5f-62aeb93b1b58.png">

<img width="814" alt="Screen Shot 2021-06-21 at 4 39 27 PM" src="https://user-images.githubusercontent.com/704760/122825352-d0419200-d2af-11eb-9d97-476ea844d02c.png">
